app/src/main/java/com/vinh/dictionary_1/screen/splash/SplashPresenter.java

```diff
@@ -1,10 +1,13 @@
 package com.vinh.dictionary_1.screen.splash;
 
+import android.app.Activity;
```

Comment:
ERROR: Unused import: android.app.Activity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports